tool/pom.xml

```diff
@@ -44,7 +44,7 @@
 		<dependency>
 			<groupId>com.ibm.icu</groupId>
 			<artifactId>icu4j</artifactId>
-			<version>58.2</version>
+			<version>61.1</version>
 		</dependency>
```

Comment:
:bulb: Ideally before this line you'd have the following:

```xml
<scope>provided</scope>
```

Then the other packaging exclusion could be completely removed:
https://github.com/antlr/antlr4/blob/e05a6a6f53c8339581523850c089f2030da68179/tool/pom.xml#L117-L121

The current update is fine, but this pair of changes would further isolate downstream dependencies from issues like the CVEs that led to this pull request.

Reply:
Thanks for the tip. Any idea when 4.7.2 will be released?

Reply:
Release scheduling is outside by area of influence 😄

Reply:
Yep! 😄

Reply:
Hi guys. So is this ready or do we need the `scope` thing @sharwell mentioned?

Reply:
What are the issues potentially with changing the version to 61.1?